backend/src/main/resources/db/migration/V1__Create_first_scheme.sql

```diff
@@ -17,3 +17,11 @@ create table questions
     eng         text                not null,
     primary key (id)
 );
+
+create table answers
+(
+    question_id varchar(36) not null references questions (id) on delete cascade,
```

Comment:
No, osobne radeji vsude cpu id a `(question_id, patient_id)` at je unique. Ale uz nevim, jestli je to BP nebo jen zvyk 🤔

Reply:
to by ti Svoboda dal 😂